rocket-chat-android-widgets/src/main/res/layout/message_inline_attachment.xml

```diff
@@ -93,14 +96,12 @@
 
         </LinearLayout>
 
-        <ImageView
+        <com.facebook.drawee.view.SimpleDraweeView
                 android:id="@+id/image"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:maxHeight="200dp"
+                android:layout_width="match_parent"
+                android:layout_height="200dp"
```

Comment:
The image is unexpectedly collapsed with setting `layout_height=wrap_content` ...
Hmm...